src/products/README.md

```diff
@@ -40,3 +40,39 @@ Updates the current inventory value for the product identified by `{productID}`.
 Returns details on all categories.
 ### GET /categories/id/{categoryID}
 Returns details on the category identified by `{categoryID}`.
+
+## Testing
+To run integration tests for the Products service a Python virtual environment is required. You must have Python 3.8+ installed on your system to run the commands below. The commands are written to be ran from the test directory of the products service (`src/products/test`).
+
+The following command will create a virtual environment. 
```

Comment:
do we have to have a virtual env for each services?

Reply:
Yes, it's being done in order to keep the tests as standalone services.